easybuild/easyblocks/n/namd.py

```diff
@@ -141,6 +143,9 @@ def configure_step(self):
             'parallel': self.cfg['parallel'],
         }
         charm_subdir = '.'.join(os.path.basename(self.charm_tarballs[0]).split('.')[:-1])
+        # From version 3.0 naming schema changed
+        if LooseVersion(self.version) >= LooseVersion('3.0'):
+           charm_subdir = charm_subdir.replace("7.0.0", "v7.0.0")
```

Comment:
Hardcoding this to 7.0.0 seems like it's begging to break on the very next release.

Reply:
I agree, I am not sure what would be an alternative way out. This just resolved the issue and we were able to successfully install the latest NAMD version with charm (the issue was with charm though). I am happy to make any further changes to the code.

Reply:
Looks like the non-beta release (Version 3.0 (2024-06-14) Platforms) on https://www.ks.uiuc.edu/Development/Download/download.cgi is now having charm-8.0.0.tar, and inside that tar, the folder name is now back to charm-8.0.0 (i.e. no "v"). So there might be no need for this easyblock patch if you're compiling the non-beta release.